app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -100,6 +100,7 @@ public class UploadActivity extends BaseActivity implements UploadContract.View,
     private Place place;
     private LatLng prevLocation;
     private LatLng currLocation;
+    public static boolean uploadIsOnPlace = false;
```

Comment:
Would it be possible to avoid a public member? For instance by making it private and adding a setter.
And maybe rename to `uploadIsOfAPlace`, I think it is more understandable.
Also, please add a Javadoc to this member (or to the setter if you write one), thanks!